Oops, looks like the udev changes broke zigbee2mqtt on non linux systems!

```
zigbee2mqtt:info  2019-10-23T20:21:51: Logging to directory: '/opt/zigbee2mqtt/data/log/2019-10-23.20-21-51'
zigbee2mqtt:info  2019-10-23T20:21:51: Starting zigbee2mqtt version 1.6.0 (commit #c18e9fb)
zigbee2mqtt:info  2019-10-23T20:21:51: Starting zigbee-herdsman...
zigbee2mqtt:error 2019-10-23T20:21:51: Error while starting zigbee-herdsman
zigbee2mqtt:error 2019-10-23T20:21:51: Failed to start zigbee
zigbee2mqtt:error 2019-10-23T20:21:51: Exiting...
zigbee2mqtt:error 2019-10-23T20:21:51: Error: spawn udevadm ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:232:19)
    at onErrorNT (internal/child_process.js:407:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zigbee2mqtt@1.6.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the zigbee2mqtt@1.6.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /opt/zigbee2mqtt/data/.npm/_logs/2019-10-23T20_21_51_861Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
     "rimraf": "*",
     "semver": "*",
     "winston": "*",
-    "zigbee-herdsman": "0.11.0",
+    "zigbee-herdsman": "0.11.1",
     "zigbee-herdsman-converters": "11.1.28"
   },
   "devDependencies": {

--- a/test/bridgeConfig.test.js
+++ b/test/bridgeConfig.test.js
@@ -237,7 +237,6 @@ describe('Bridge config', () => {
         expect(device.removeFromNetwork).toHaveBeenCalledTimes(1);
         expect(controller.state[device.ieeeAddr]).toBeUndefined();
         expect(settings.getDevice('bulb_color')).toBeNull();
-        expect(MQTT.publish).toHaveBeenCalledTimes(1);
         expect(MQTT.publish).toHaveBeenCalledWith(
             'zigbee2mqtt/bridge/log',
             JSON.stringify({type: 'device_removed', message: 'bulb_color'}),

